src/simulation/sensors/camera/camera.h

```diff
@@ -52,44 +52,43 @@ class Camera: public SysModel {
-    int32_t saveImages;                  //!< [-] 1 to save images to file for debugging
+    std::string saveDir{};                 //!< The name of the directory to save images
+    uint64_t sensorTimeTag{};              //!< [ns] Current time tag for sensor out
+    int32_t saveImages{};                  //!< [-] 1 to save images to file for debugging
     
     /*! Camera parameters */
-    int cameraIsOn; //!< [-] Is the camera currently taking images
-    int cameraID; //!< [-] Is the camera currently taking images
-    int resolution[2];         //!< [-] Camera resolution, width/height in pixels (pixelWidth/pixelHeight in Unity) in pixels
-    uint64_t renderRate;       //!< [ns] Frame time interval at which to capture images in units of nanosecond
-    double fieldOfView;        //!< [r] camera y-axis field of view edge-to-edge
-    double cameraPos_B[3];     //!< [m] Camera position in body frame
-    double sigma_CB[3];        //!< [-] MRP defining the orientation of the camera frame relative to the body frame
-    char skyBox[MAX_STRING_LENGTH]; //!< [-] name of skyboz in use
-    int postProcessingOn;       //!< Enable post-processing of camera image. Value of 0 (protobuffer default) to use viz default which is off, -1 for false, 1 for true
-    double ppFocusDistance;     //!< Distance to the point of focus, minimum value of 0.1, Value of 0 to turn off this parameter entirely.
-    double ppAperture;          //!<  Ratio of the aperture (known as f-stop or f-number). The smaller the value is, the shallower the depth of field is. Valid Setting Range: 0.05 to 32. Value of 0 to turn off this parameter entirely.
-    double ppFocalLength;       //!< [m] Valid setting range: 0.001m to 0.3m. Value of 0 to turn off this parameter entirely.
-    int ppMaxBlurSize;          //!< Convolution kernel size of the bokeh filter, which determines the maximum radius of bokeh. It also affects the performance (the larger the kernel is, the longer the GPU time is required). Depth textures Value of 1 for Small, 2 for Medium, 3 for Large, 4 for Extra Large. Value of 0 to turn off this parameter entirely.
-
     char parentName[MAX_STRING_LENGTH]{};  //!< [-] Name of the parent body to which the camera should be attached
+    int cameraIsOn{}; //!< [-] Is the camera currently taking images
+    int cameraID{1}; //!< [-] Is the camera currently taking images
+    int resolution[2]{512, 512};         //!< [-] Camera resolution, width/height in pixels (pixelWidth/pixelHeight in Unity) in pixels
+    uint64_t renderRate{};       //!< [ns] Frame time interval at which to capture images in units of nanosecond
+    double fieldOfView{0.7};       //!< [r] camera y-axis field of view edge-to-edge
+    double cameraPos_B[3]{};     //!< [m] Camera position in body frame
+    double sigma_CB[3]{};        //!< [-] MRP defining the orientation of the camera frame relative to the body frame
```

Comment:
Any reason why we're not using MRPd and Vector3d for ``sigma_CB`` and ``cameraPos_B``?

Reply:
The reason I haven't changed it is I was trying to restrain myself in regards to the extent of the cleans ups I did. I think they are better saved to a large rewrite of this module.